content/post/71-putting-npm-on-ipfs-part-2.md

```diff
@@ -0,0 +1,82 @@
+---
+date: 2019-02-18
+title: Putting npm on IPFS Part 2 - The Client
+author: Alex Potsides
+---
+
+In [Putting npm on IPFS Part 1 - The Registry]((/post/70-putting-npm-on-ipfs-part-1)) we looked at [registry.js.ipfs.io](https://registry.js.ipfs.io) - a mirror of the npm registry that adds [CID](https://docs.ipfs.io/guides/concepts/cid)s to package metadata and uses IPFS to fetch tarballs using IPFS.
+
+Using the registry in this way works, but it still depends on HTTP and DNS to resolve your dependencies. So far so centralised.
+
+What if there was a way to skip that bit and have it be distributed-web-turtles all the way down?
```

Comment:
Would be great to mention the benefits that using npm on IPFS brings here

Reply:
ex at least some of: resiliency from central failures, lower central bandwidth costs, faster installation/updates, less duplication on disk, etc?

Reply:
Done!